<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelftestWin</class>
 <widget class="QWidget" name="SelftestWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelftestWin</string>
  </property>
  <widget class="QPushButton" name="pbValve">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Valve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMotor">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Selftest</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSensor">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbVersion">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOther">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
